src/main/java/org/broadinstitute/hellbender/tools/pon/allelic/AllelicPanelOfNormals.java

```diff
@@ -0,0 +1,331 @@
+        Utils.nonNull(outputFile);
+        final List<Map.Entry<SimpleInterval, HyperparameterValues>> sortedMapEntries = collectSortedMapEntries(siteToHyperparameterValuesMap);
+        try (final AllelicPanelOfNormalsWriter writer = new AllelicPanelOfNormalsWriter(outputFile)) {
+            writer.writeComment(GLOBAL_ALPHA_COMMENT_STRING + AllelicPanelOfNormalsWriter.formatDouble(globalHyperparameterValues.alpha));
+            writer.writeComment(GLOBAL_BETA_COMMENT_STRING + AllelicPanelOfNormalsWriter.formatDouble(globalHyperparameterValues.beta));
+            writer.writeAllRecords(sortedMapEntries);
+        } catch (final IOException e) {
+            throw new UserException.CouldNotCreateOutputFile(outputFile, e);
+        }
+    }
+
+    /**
+     * Writes out the {@link AllelicPanelOfNormals} to the specified HDF5 file with the specified {@link HDF5File.OpenMode}.
+     * See {@link HDF5AllelicPoNUtils#write}.
+     * @param outputHDF5File    HDF5 file to write to (if it exists, it will be overwritten)
+     * @param openMode          desired {@link HDF5File.OpenMode} (if {@code HDF5File.OpenMode.READ_ONLY}, an exception will be thrown)
+     */
+    public void write(final File outputHDF5File, final HDF5File.OpenMode openMode) {
+        Utils.nonNull(outputHDF5File);
+        try (final HDF5File hdf5File = new HDF5File(outputHDF5File, openMode)) {
+            HDF5AllelicPoNUtils.write(hdf5File, this);
+        }
+    }
+
+    public static class HyperparameterValues {
+        private final double alpha;
+        private final double beta;
+
+        public HyperparameterValues(final double alpha, final double beta) {
+            this.alpha = alpha;
+            this.beta = beta;
+        }
+
+        /**
+         * Initializes the hyperparameter values at a site given the total counts observed across all normals.
+         * @param alpha MLE global hyperparameter value for alpha
+         * @param beta  MLE global hyperparameter value for beta
+         * @param a     total alt counts observed across all normals at site
+         * @param r     total ref counts observed across all normals at site
+         */
+        private HyperparameterValues(final double alpha, final double beta, final int a, final int r) {
+            final double f = 0.5;
+            final int n = a + r;
+            final double lambda0 = AlleleFractionLikelihoods.biasPosteriorMode(alpha, beta, f, a, r);
+            final double kappa = AlleleFractionLikelihoods.biasPosteriorCurvature(alpha, f, r, n, lambda0);
+            this.alpha = AlleleFractionLikelihoods.biasPosteriorEffectiveAlpha(lambda0, kappa);
+            this.beta = AlleleFractionLikelihoods.biasPosteriorEffectiveBeta(lambda0, kappa);
+        }
+
+        public double getAlpha() {
+            return alpha;
+        }
+
+        public double getBeta() {
+            return beta;
+        }
+    }
+
+    private static boolean isHDF5File(final File inputFile) {
```

Comment:
Could this be delegated to one of the classes that deals with HDF5 specifically? Seems like some other classes might want to use it